src/Microsoft.Diagnostics.EventFlow.ServiceFabric/ServiceFabricDiagnosticPipelineFactory.cs

```diff
@@ -14,6 +17,9 @@ namespace Microsoft.Diagnostics.EventFlow.ServiceFabric
 {
     public static class ServiceFabricDiagnosticPipelineFactory
     {
+        public static readonly string ConfigurationPackageName = "Config";
```

Comment:
That is good suggestion. I decided to use the definition in ServiceFabricConfigurationProvider